kansha/batch/create_index.py

```diff
@@ -27,17 +27,23 @@
 
 import pkg_resources
 
-from nagare.admin import util, command
 from nagare import database
+from nagare.admin import util, command
 
-from kansha.card.fts_schema import Card as SCard
+from kansha.card.comp import Card
+from kansha.column.comp import Column
 from kansha.card.models import DataCard
+from kansha.card.fts_schema import Card as SCard
+from kansha.services.actionlog import DummyActionLog
 
 
 def rebuild_index(app):
     # init index
+    action_log = DummyActionLog()
     app.search_engine.create_collection([SCard])
-    for card in DataCard.get_all():
+    for card in DataCard.query:
+        col = app._services(Column, card.column.id, card.column.board, app.card_extensions, action_log, app.search_engine, data=card.column)
+        card = app._services(Card, card.id, col, app.card_extensions, action_log, data=card)
         scard = SCard.from_model(card)
```

Comment:
Let's drop `from model` and explicitely instanciate SCard(**card.to_document()).